csharp/src/Ice/LocalException.cs

```diff
@@ -85,51 +85,83 @@ public NoEndpointException(string stringifiedProxy)
     /// <summary>This exception reports an error from the transport layer.</summary>
     public class TransportException : Exception
     {
+        /// <summary>The exception retry policy.</summary>
+        public RetryPolicy RetryPolicy { get; }
```

Comment:
I believe exceptions with field/properties need custom serialization.

Reply:
Do we care about binary serialization anymore, we removed it because .NET now warns against using it because of the security implications.